documentation/bolt.ditamap

```diff
@@ -32,7 +32,6 @@
     <topicref href="bolt_configuration_reference.md" format="markdown"/>
     <topicref href="bolt_configure_orchestrator.md" format="markdown"/>
     <topicref href="bolt_connect_puppetdb.md" format="markdown"/>
-    <topicref href="inventory_file.md" format="markdown"/>
```

Comment:
Should we delete this file altogether?

Reply:
Wasn't sure if we wanted to keep it for a potential 'archive' or not, but I guess we decided we're not supporting old docs? Removed.